tools/updateIoRangeSlider.R

```diff
@@ -0,0 +1,69 @@
+tmpdir <- tempdir()
+
+# https://github.com/IonDen/ion.rangeSlider
+version <- "2.3.1"
+zip_src <- sprintf("https://github.com/IonDen/ion.rangeSlider/archive/%s.zip", version)
+zip_target <- file.path(tmpdir, "ion.zip")
+download.file(zip_src, zip_target)
+unzip(zip_target, exdir = dirname(zip_target))
+src <- file.path(dirname(zip_target), paste0("ion.rangeSlider-", version))
+target <- "inst/lib/ionrangeslider"
+unlink(target, recursive = TRUE)
+dir.create(target)
+# Move over JS files
+file.rename(
+  file.path(src, "js"),
+  file.path(target, "js")
+)
```

Comment:
Are you still planning on using this build script to update the ion range slider dependency? I'm pretty sure this will require changes to the markup that `filter_slider()` produces, so how about we do that in a follow up PR?

Reply:
I did not really think this through and blindly updated it; though it seems to work it's likely to break things. I'll remove this and will do in another PR.